tests/sans2d_vacuum_collision_avoidance.py

```diff
@@ -73,7 +91,6 @@ def __repr__(self):
              name="RBRD", interval_setpoint_name="RBSPRDSP", minimum_interval=350),
 ]
 
-
 BAFFLES_AND_DETECTORS_Z_AXES = set(
     [interval.front_axis for interval in AXIS_PAIRS] + [interval.rear_axis for interval in AXIS_PAIRS])
 
```

Comment:
I don't think `MAJOR_ALARM_INTERVAL_THRESHOLD` and `MINOR_ALARM_INTERVAL_THRESHOLD` are used anymore

Reply:
Removed